logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -167,4 +152,10 @@ internal class ConversationMapperImpl(
             -> PersistedConversation.Type.ONE_ON_ONE
         }
     }
+
+    private fun ConversationEntity.Type.fromDaoModelToType(): Conversation.Type = when (this) {
```

Comment:
Since DB models have a different name now we can delete the aliases in the import (line 18, 19)